exonum-java-binding/core/rust/src/storage/proof_map_index.rs

```diff
@@ -446,13 +449,17 @@ pub extern "system" fn Java_com_exonum_binding_core_storage_indices_ProofMapInde
     key: jbyteArray,
```

Comment:
xFrom have never been supported in Java. We could add support for them, but shall we (in any foreseeable future) — @slowli ? If I understand correctly, they require the user to be aware of how keys are serialized and how that results in a particular ordering.

Reply:
https://jira.bf.local/browse/ECR-3942